tools/pkglint/lib/rules.ts

```diff
@@ -218,14 +219,41 @@ function cdkModuleName(name: string) {
     name = name.replace(/^aws-cdk-/, '');
     name = name.replace(/^@aws-cdk\//, '');
 
+    const dotnetSuffix = name.split('-').map(s => s === 'aws' ? 'AWS' : caseUtils.pascal(s)).join('.');
```

Comment:
use some new lines to make this easier to follow

Reply:
✔️